### PR TITLE
Move TestPropertiesTest to new folder

### PR DESCRIPTION
**Pull request by roncain**
The new TestProperties feature had a unit test to
validate the code generation worked properly.
But the test for it was in the wrong place and
could not be run under the experimental NET Native
test environment.

This change creates a new folder and project specifically
for testing the common test infrastructure itself, and it
moves TestPropertiesTest into it.

### CODE REVIEW

File: src/System.Private.ServiceModel/tests/Common/tests/project.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "System.Xml.XmlSerializer": "4.0.10-beta-*",
+    "xunit": "2.0.0-beta5-build2785",
+    "xunit.abstractions.netcore": "1.0.0-prerelease",
+    "xunit.assert": "2.0.0-beta5-build2785",
+    "xunit.core.netcore": "1.0.1-prerelease"
+  },
+  "frameworks": {
+    "dnxcore50": {}
+  }
+}
```
Comment (SajayAntony) [inline src/System.Private.ServiceModel/tests/Common/tests/project.json]: Nit: newline fix

File: src/System.Private.ServiceModel/tests/Common/tests/System.ServiceModel.Common.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      <Project>{9e50e7bf-cd6e-4269-a6dd-59fd0bd6c0fd}</Project>
+      <Name>System.Private.ServiceModel</Name>
+    </ProjectReference>
+    <ProjectReference Include='$(WcfTestCommonProj)'>
+      <Project>{e896294a-ab4a-4af5-a01c-a19e3972eff9}</Project>
+      <Name>System.ServiceModel.Tests.Common</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+</Project>
```
Comment (SajayAntony) [inline src/System.Private.ServiceModel/tests/Common/tests/System.ServiceModel.Common.Tests.csproj]: Same (new line fix);

**Comment by SajayAntony** [comment]
:ship:

**Comment by StephenBonikowsky** [comment]
:shipit:

**Pull request merged**